### ISSUE TITLE
Possibility of generating Sawtooth waves with device

### ISSUE THREAD

**Issue by mariobehling** [fossasia/pslab-hardware#109]
Would it be possible to generate sawtooth waves with the device additionally to sin and square waves?

**Comment by CloudyPadmal** [comment]
Yes, sawtooth waves were there in the older desktop app. Let me see what has happened to the feature :thinking:
